index.js

```diff
@@ -86,18 +86,13 @@ export const step = {
   stepHorizontal: 'group/steph grid-rows-[auto_20px] grid-cols-[1fr_20px_1fr] flex-1 grid gap-y-16 items-center',
 
   stepDot: 'rounded-full border-2 h-20 w-20 transition-colors duration-300 i-text-$color-stepindicator-handle-icon',
-  stepDotVertical: '',
-  stepDotVerticalLeft: '',
   stepDotVerticalRight: 'col-start-2',
   stepDotHorizontal: 'row-start-2 justify-self-end',
-
   stepDotActive: 'i-border-$color-stepindicator-handle-border-active i-bg-$color-stepindicator-handle-background-active',
-  stepDotComplete: 'i-border-$color-stepindicator-handle-border-active i-bg-$color-stepindicator-handle-background-active',
```

Comment:
I don't think this should be removed 🤔 What was the reason for this change?

Reply:
It was a duplicate of the `stepDotActive`, as they contain the same styling

Reply:
Ah, sorry then 🙃